index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles/style.css">
+    <title>Home</title>
+  </head>
+
+  <body>
+    <header class="head">
+      <div class="checkout">
+        <a href="checkout.html">
+          <img src="images/cart-icon.svg" alt="Cart Icon" >
+          $1234.56
+        </a>
+        <input type="button" name="cart" value="checkout">
+      </div>
+      <img src="images/adoptadog-logo.svg" alt="Adoptadog logo">
+      <p> Seattle’s premiere dog adoption agency </p>
+
+    </header>
+
+    <nav class="top">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="doglist.html">Dogs</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </nav>
+
+    <div class="top-section">
+      <div class="top-image">
```

Comment:
If you only have an image in a div, you might not need the wrapping div. It depends on what you're doing of course, but arguably you don't need it.